infrastructure/resource_group_level/modules/redis.bicep

```diff
@@ -0,0 +1,42 @@
+param location string
+param environment string
+
+var redisName = 'tfs-${environment}-redis'
+
+// See https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/cache-redis-cache-bicep-provision?tabs=CLI
```

Comment:
I could add some of the extras in their example. but it doesn't seem especially necessary.